ydb/docs/en/core/recipes/ydb-sdk/session-pool-limit.md

```diff
@@ -2,13 +2,15 @@
 
 {% include [work in progress message](_includes/addition.md) %}
 
-The client's session pool size affects resource consumption (RAM, CPU) on the server side of {{ ydb-short-name }}. Simple math: if `1000` clients of the same DB have `1000` sessions each, `1000000` actors (workers, session performers) are created on the server side. If you don't limit the number of sessions on the client, this may result in a slow cluster that is close to a failure.
+{{ ydb-short-name }} creates an [actor](../../concepts/glossary.md#actor) for each session. Consequently, session pool size of a client affects resource consumption (RAM, CPU) on the server side of {{ ydb-short-name }}.
 
-By default, the {{ ydb-short-name }} SDK has a limit of `50` sessions when using native drivers. There is no limit for third-party libraries, such as Go database/sql.
+For example, if 1000 clients of the same database have 1000 sessions each, 1000000 [actors](../../concepts/glossary.md#actor) are created on the server side, and this number of actors consumes a significant amount of memory and CPU. If you do not limit the number of sessions on the client, this may result in a slow cluster that is close to a failure.
```

Comment:
```suggestion
For example, if 1000 clients of the same database have 1000 sessions each, then 1000000 [actors](../../concepts/glossary.md#actor) are created on the server side, consuming a significant amount of memory and CPU. If you do not limit the number of sessions on the client side, this may result in a slow cluster that is at risk of failure.
```